contrib/ansible/README.md

```diff
@@ -9,6 +9,7 @@ can be real hardware, VMs, things in a public cloud, etc.
 * Record the IP address/hostname of which machine you want to be your master (only support a single master)
 * Record the IP address/hostname of the machine you want to be your etcd server (often same as master, only one)
 * Record the IP addresses/hostname of the machines you want to be your nodes. (the master can also be a node)
+* Make sure your ansible running machine has ansible and python-netaddr installed.
```

Comment:
suggesting to removing that netaddr dependency ? that  could simplify things for folks who dont have a heavily pythonified ansible provisioner  .  but i dont think its a huge problem.